admin/apple-actions/index/class-export.php

```diff
@@ -247,6 +259,7 @@ public function fetch_exporter() {
 			$post->ID,
 			$title,
 			$content,
+            $slug,
```

Comment:
Spacing is off here.
```suggestion
			$slug,
```